src/gui/editors/PianoRoll.cpp

```diff
@@ -485,11 +494,21 @@ void PianoRoll::changeNoteEditMode( int i )
 }
 
 
-void PianoRoll::markSemiTone( int i )
+/*! \brief highlight/clear piano key row or rows based on action i
```

Comment:
As Dom has already noted, these would be better in the header.

Reply:
Yeah, I agreed with it. :)